cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -158,8 +217,12 @@ namespace lexicographic {
  * `aac < abb`.
  *
  * @tparam Nullate A cudf::nullate type describing whether to check for nulls.
+ * @tparam PhysicalElementComparator A relational comparator functor that compares individual values
+ * rather than logical elements, defaults to `NaN` aware relational comparator that evaluates `NaN`
+ * as greater than all non-null values.
```

Comment:
The responsibility of handling nulls doesn't actually go to the physical comparator. I wonder if we should not mention that here at all.